doc/modules/gaussian_process.rst

```diff
@@ -610,7 +610,7 @@ shown in the following figure:
 References
 ----------
 
-.. [RW2006] Carl Eduard Rasmussen and Christopher K.I. Williams, "Gaussian Processes for Machine Learning", MIT Press 2006, Link to an official complete PDF version of the book `here <http://www.gaussianprocess.org/gpml/chapters/RW.pdf>`_ .
+.. [RW2006] Carl Eduard Rasmussen and Christopher K.I. Williams, "Gaussian Processes for Machine Learning", MIT Press 2006, `Book link <http://www.gaussianprocess.org/gpml/chapters/RW.pdf>`_
```

Comment:
If you can avoid a long line here and do a multiline thing as in other places, that would be great!

Reply:
Sure, will do. I just adapted the already present line in `modules/gaussian_process.rst`. I did the conversion to multi-line in the GPR and GPC doc strings.

Reply:
I will adapt the refs to use a full link as you suggest above. 

In `modules/gaussian_process.rst` the ref is called `[RW2006]`. In doc strings they were numbered. What's the policy here? I changed them to `[RW2006]` in order to be consistent with the user guide.